django/contrib/postgres/search.py

```diff
@@ -138,8 +138,8 @@ def _combine(self, other, connector, reversed):
                 'instances, got %s.' % type(other).__name__
             )
         if reversed:
-            return CombinedSearchQuery(other, connector, self, self.config)
-        return CombinedSearchQuery(self, connector, other, self.config)
```

Comment:
Combined `SearchQuery`'s should have configs, I added a new test in e2417010daf697d5a58bc2057a24b7785c4cef03 to prevent future regressions.